SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -273,14 +270,15 @@ def customise_harvesting(process):
     process.DQMOffline_SecondStep_PrePOG.remove(process.normalizePlotsRealMuonsData)
     process.postValidation.remove(process.runTauEff)
 
+    # TODO this should be removed by the era switch, but I can't find where it comes from
+    process.dqmHarvesting.remove(process.sipixelCertification)
+

```

Comment:
As I mentioned in https://github.com/cms-sw/cmssw/pull/13987#discussion_r59003111, AFAICT `sipixelCertification` is inserted to sequences in here https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/DQMOffline/Configuration/python/DQMOffline_CRT_cff.py#L18 and here https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/DQMOffline/Configuration/python/DQMOffline_Certification_cff.py#L13.
Please move the removal to be done there with eras before integration.

Out of curiosity, does the `sipixelCertification` crash or something when the phase1 pixel DQM is enabled?